content/applications/hr/employees/new_employee.rst

```diff
@@ -0,0 +1,259 @@
+  this process for the :guilabel:`Levels` section. Click :guilabel:`Add a line` to add each level
+  and progress.
+
+  .. image:: new_employee/new-skills.png
+     :align: center
+     :alt: Add a new skill and levels.
+
+  .. example::
+     To add a math skill set, enter `Math` in the :guilabel:`Skill Type` field. In the
+     :guilabel:`Skills` field, enter `Algebra`, `Calculus`, and `Trigonometry`. And, in the
+     :guilabel:`Skill Level` field enter `beginner`, `intermeidate`, and `expert`. Then, either
+     click :guilabel:`Save & Close` of :guilabel:`Save & New`.
+
+- :guilabel:`Skill`: The corresponding :guilabel:`Skills` associated with the selected
+  :guilabel:`Skill Type` appear in a drop-down menu. For example, selecting *language* as the
+  :guilabel:`Skill Type` presents a variety of languages to select from under the
+  :guilabel:`Skills` field. Select the appropriate pre-configured skill, or type in a new one.
+- :guilabel:`Skill Level`: Pre-defined :guilabel:`Skill Levels` associated with the selected skill
+  type appear in a drop-down menu. Select a level or create a new skill level by typing it in.
+- :guilabel:`Progress`: Progress is automatically selected based on the selected :guilabel:`Skill
+  Level`. Skill levels and progress can be modified in the :guilabel:`Skill Type` pop-up form, which
+  is accessed via the :guilabel:`External Link` icon next to :guilabel:`Skill Type` field.
+
+To delete any line from the :guilabel:`Resume` tab, click the :guilabel:`🗑️ (trash can)` delete
+icon to delete the entry. Add a new line by clicking the :guilabel:`ADD` button next to the
+corresponding section.
+
+Work information tab
+--------------------
+
+- :guilabel:`Location`: Select the :guilabel:`Work Address` from the drop-down menu. The
+  :guilabel:`External Link` icon opens up the selected company form in a window, and allows for
+  editing. The :guilabel:`Work Location` is where any specific location details should be noted,
+  such as a floor, or building.
+- :guilabel:`Approvers`: Using the drop-down menus, select the employees responsible for approving
+  :guilabel:`Time Off`, :guilabel:`Expenses`, and :guilabel:`Timesheets` for the employee. The
+  :guilabel:`External Link` icon opens a form with the approver's :guilabel:`Name`, :guilabel:`Email
+  Address`, :guilabel:`Company`, :guilabel:`Phone`, and :guilabel:`Mobile` fields. These can be
+  modified, if needed. Click :guilabel:`Save` after making any edits.
+- :guilabel:`Schedule`: Select the :guilabel:`Working Hours` (required) and :guilabel:`Timezone` for
+  the employee. The :guilabel:`Working Hours External Link` icon opens up a detailed view of the
+  specific daily working hours. Working hours can be created, modified, or deleted here.
+  :guilabel:`Global Time Off` (such as holidays) can be entered in the :guilabel:`Global Time Off`
+  tab. Click :guilabel:`Add a line` to add a new global time off.
```

Comment:
Keeping consistency in referring to **External Link** icon and removing the :guilabel: from the statement of _Global time off_ (such as holidays)...
```suggestion
- :guilabel:`Schedule`: Select the :guilabel:`Working Hours` (required) and :guilabel:`Timezone` for
  the employee. The :guilabel:`External Link` icon opens up a detailed view of the specific daily
  working hours. Working hours can be created, modified, or deleted here. Global time off (such as
  holidays) can be entered in the :guilabel:`Global Time Off` tab. Click :guilabel:`Add a line` to
  add a new global time off.
```